apps/web-giddh/src/app/inventory/inventory.component.scss

```
@@ -82,4 +82,38 @@
         width: 100%;
     }
 }
+@media(max-width:767px){
+    .inventory-controll ::ng-deep.nav.nav-tabs {
+        position: fixed;
+        width: 100%;
+        top: 100px;
+    }
+}
+@media(max-width:1024px){
+    .wrap-top-inventory{
+        display: flex;
+    }
+}
+
+
+@media (max-width:445px){
+.inventory-controll .container-fluid {
```

Do use correct spelling
```suggestion
.inventory-control .container-fluid {
```

---

Done